zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -98,6 +98,39 @@ paths:
             The assignment of the user task with the given key cannot be completed.
             More details are provided in the response body.
           $ref: "#/components/responses/ProblemResponse"
+  /user-tasks/{userTaskKey}/update:
```

Comment:
>  /user-tasks/{userTaskKey}/update:

@tmetzke Can we please change this to `PATCH /user-tasks/{userTaskKey}`? That's how it's currently implemented right?

Reply:
Holy guacamole 🥑, how did nobody see this before? 🙈 
Yes, I'll adjust the spec to `PATCH /user-tasks/{userTaskKey}` 👍

Reply:
I noticed it in the REST explorer :D

Reply:
Good thing we're finally generating human-readable documentation 👍

Reply:
https://github.com/camunda/zeebe/issues/16929